src/Controller/Admin/UserController.php

```diff
@@ -1154,4 +1202,34 @@ private function generateCustomUrl(array $params, string $fallbackUrl = 'pimcore
 
         return $loginUrl;
     }
+
+    private function isComplexPassword(string $password): bool
```

Comment:
Maybe add a documentation comment or rename in a way similar to `bsi_standard_*` prefixes used elsewhere to show that BSI has non-standardised password rules:

```suggestion
    private function isBsiComplexPassword(string $password): bool
```

Reply:
I have renamed the function names